enhancements/update/duration-of-migration-to-multi-arch.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Duration of Migration To Multi Arch
```

Comment:
[linter][1]:

```
enhancements/update/duration-of-migration-to-multi-arch.md: the title Duration of Migration To Multi Arch and the file base name duration-of-migration-to-multi-arch must match
```

and [the template has][2]:

```yaml
title: neat-enhancement-idea
```

so I suspect this line should be:

```yaml
title: duration-of-migration-to-multi-arch
```

[1]: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_enhancements/1690/pull-ci-openshift-enhancements-master-markdownlint/1847350387175067648#1:build-log.txt%3A532
[2]: https://github.com/openshift/enhancements/blob/7d163511f45e73e707f8ec17f34a446772b2cff6/guidelines/enhancement_template.md?plain=1#L2

Reply:
The job is green now.